tests/unit/test_routes.py

```diff
@@ -594,6 +594,47 @@ def test_search(module_client, module_db, fake_auth_from_oc, fake_algolia_save,
     assert (result.json['data'][0]['url'] == resource.json['data'].get('url'))
 
 
+def test_search_paid_filter(module_client, module_db, fake_auth_from_oc, fake_algolia_save, fake_algolia_search):
+    client = module_client
+
+    apikey = get_api_key(client)
+
+    # Create paid and unpaid resource to find it in the search results.
+    paid_resource = client.post("/api/v1/resources",
+                          json=dict(
+                              name=f"Tester1",
```

Comment:
Not that it's a problem, but why have these be format strings rather than just a regular string?

Reply:
Wops, looks like I missed that 😅